src/test/rpc/LedgerData_test.cpp

```diff
@@ -344,84 +354,91 @@ class LedgerData_test : public beast::unit_test::suite
         auto const jrr = makeRequest(jss::account);
         BEAST_EXPECT( checkArraySize(jrr[jss::state], 12) );
         for (auto const& j : jrr[jss::state])
-            BEAST_EXPECT( j["LedgerEntryType"] == "AccountRoot" );
+            BEAST_EXPECT( j["LedgerEntryType"] == jss::AccountRoot );
```

Comment:
should `"LedgerEntryType"` also be in `jss` ?

Reply:
`LedgerEntryType` is one of the serialized fields (defined in SField.cpp).  We should probably getting the string from `sfLedgerEntryType.fieldName`.

My eventual hope is that we get any strings that are in an `SField` from there.  All other strings should come from `jss`.  But I'm a long way from achieving that.  For now I'm limiting myself to transaction and ledger types.  This pull request is just a step along the path.